packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -46,40 +71,73 @@ class AccordionTitle extends UIComponent<WithAsProp<AccordionTitleProps>, any> {
 
   static className = 'ui-accordion__title'
 
+  static slotClassNames: AccordionTitleSlotClassNames
+
   static propTypes = {
     ...commonPropTypes.createCommon(),
     active: PropTypes.bool,
-    index: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    buttonRef: customPropTypes.ref,
```

Comment:
let's rename this to contentRef, as is no longer a Button

Reply:
let's think about it more. I also added contentId to point to the AccordionContent. I think contentRef will create some confusion. What do you think?